### PR TITLE
Bug auto download broken 111

### PR DESCRIPTION
**Pull request by poshi1865**
#111 Checked whether OS is windows or not, and gave paths accordingly. Please review and let me know.

### CODE REVIEW

**Comment by poshi1865** [comment]
Oh sorry, I think this pr contains my commits done in the previous fix, let me work on fixing #1 first, and then this will become mergeable as well.

**Comment by SaptarshiSarkar12** [comment]
> Oh sorry, I think this pr contains my commits done in the previous fix, let me work on fixing #1 first, and then this will become mergeable as well.

Ok @poshi1865

**Comment by poshi1865** [comment]
I think this is now mergeable too @SaptarshiSarkar12

**Comment by SaptarshiSarkar12** [comment]
@poshi1865 Please resolve the branch conflicts.

**Comment by poshi1865** [comment]
the conflicting file is a .class file, and it is showing that I cannot resolve the conflict, only people with write access can

**Comment by SaptarshiSarkar12** [comment]
> the conflicting file is a .class file, and it is showing that I cannot resolve the conflict, only people with write access can

ok @poshi1865 , I have fixed it.

**Comment by SaptarshiSarkar12** [comment]
Sorry @poshi1865 , closed it by mistake

File: src/Drifty_CLI.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import java.awt.*;
```
Comment (SaptarshiSarkar12) [inline src/Drifty_CLI.java:1]: @poshi1865 Is `awt` package required in this program?

Comment (poshi1865) [reply src/Drifty_CLI.java:1]: Umm I don't think so, I did not add any awt classes, let me check once.

Comment (SaptarshiSarkar12) [reply src/Drifty_CLI.java:1]: @poshi1865 Please respond.

Comment (poshi1865) [reply src/Drifty_CLI.java]: I resolved it @SaptarshiSarkar12 , it was unused.

**Review by SaptarshiSarkar12 — APPROVED** [review]
LGTM 👍. Thanks for contributing 🚀🚀.

File: src/Drifty_CLI.java (hunk 1 above)
Comment (SaptarshiSarkar12) [reply src/Drifty_CLI.java]: ok @poshi1865

**Pull request merged**